neuralcompression/layers/_rate_distortion_loss.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import math
+from typing import List, NamedTuple
+
+import torch
+from torch import Tensor
+from torch.nn import Module, MSELoss
+
+
+class _ForwardReturnType(NamedTuple):
+    bpp: float
+    mse: float
+    rate_distortion: float
```

Comment:
Could we make this public? And maybe change the name to `RDLoss` or something.

Reply:
Yep. I refactored this a bit to make it more accessible.